ibm/operator_collection_sdk/README.md

```diff
@@ -204,22 +204,19 @@ Enter your ZosEndpoint name:
 [kubernetes]:https://github.com/kubernetes-client/python#installation
 [broker]:https://ibm.biz/ibm-zoscb-install
 
-## Configure extra-vars file to bypass prompts
-If you find yourself inputting vars_prompts frequently, append `--extra-vars <vars>.json` to your playbook command/alias
-
-```bash
-ansible-playbook create_credential_secret.yml --extra-vars vars.json
-```
-
-Example `vars.json` file:
-
-```
-{
-    username: "testuser",
-    operator_name: "racf",
-    ssh_key: "~/.ssh/id_rsa",
-    secret_name: "test-secret"
-}
+## Configure ocsdk-extra-vars file to bypass prompts
+If you find yourself inputting vars_prompts frequently, create an `ocsdk-extra-vars.yaml` file. If `create_operator.yml` detects this file, it will automatically use it.
+
+Example `ocsdk-extra-vars.yaml` file:
+
+```yaml
+zosendpoint_type: remote
+zosendpoint_name: my-endpoint
+zosendpoint_host: 1.2.3.4
+zosendpoint_port: '22'
+username: user
+ssh_key: ~/.ssh/id_rsa
+passphrase: my_ssh_passphrase
```

Comment:
Can you add a note below this snippet that it is recommended that passphrases aren't explicitly specified in this file, and to continue to provide this value view the vars prompt. If specified, then the recommendation is to add this  file to your `.gitignore` so that's it's not checked into Github.